src/react-apps/applications/shared/src/types/index.d.ts

```diff
@@ -20,20 +20,20 @@ export interface IData {
   fileName: string;
   contentType: string;
   storageUrl: string;
-  dataLinks: IDataLinks;
+  selfLinks: ISelfLinks;
```

Comment:
`selfLinks` is plural, but datatype in singular.

Reply:
What is the proposal?

Reply:
Renamed from IDataTypes to IDataType.